napari/layers/labels/_labels_mouse_bindings.py

```diff
@@ -0,0 +1,37 @@
+from ._labels_utils import interpolate_coordinates
+
+
+def paint(layer, event):
+    """Paint with the currently selected label."""
+    # on press
+    layer._save_history()
+    layer._block_saving = True
+    layer.paint(layer.coordinates, layer.selected_label)
+    last_cursor_coord = layer.coordinates
+    yield
+
+    # on move
+    while event.type == 'mouse_move':
+        interp_coord = interpolate_coordinates(
+            last_cursor_coord, layer.coordinates, layer.brush_size
+        )
+        for c in interp_coord:
+            layer.paint(c, layer.selected_label, refresh=False)
+        layer.refresh()
```

Comment:
Is this fast enough now, or is this a performance regression?

Reply:
It is, this is not changed by this PR. It still needs to be faster, but that will come :-)